posts/structural-search-patterns.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How To Search with Sourcegraph using Structural Patterns
+author: marek-zaluski
+tags: [tutorial, search, Sourcegraph]
+description: Learn how to use structural search patterns to search code on Sourcegraph.
+image: /headers/sourcegraph-learn-header-8.svg
+imageAlt: Sourcegraph Learn
+socialImage: /headers/sourcegraph-learn-header-8.png
```

Comment:
I would recommend header 10 because it's blue, otherwise we'll have two yellow ones in this series, and they sit next to each other on the homepage too.